node-operators/parachain-node/parachain-node.md

```diff
@@ -0,0 +1,237 @@
+## Requirements
+
+
+### Network 
+The relay and parachain p2p port must be open to incoming traffic from the internet.  
+```
+Source: Any
+Destination: 30333, 30334 TCP
+```
+If you are planning to run external RPC endpoints, you will need to allow those incoming ports as well.  See below for information on default ports. 
+
+### CPU
+8 Cores (early development phase - not optimized yet)
+
+### RAM
+16 GB (early development phase - not optimized yet)
+
+### SSD
+In Alphanet, this will be very modest to start, 50GB, as we are running a new relay chain.  In mainnet however, you will have to add the disk requirements of Kusama or Polkadot to that of the parachain. 
+
+
+## Install Instructions - Docker
+A Moonbase Alphanet parachain node can be spun up quickly using the docker command below.  
+
+Create a local directory to store the chain data.
+   ```
+   mkdir /var/lib/moonbase-alpha
+   ```
+Update the command below, replace `YOUR-NODE-NAME` (2 places), verify the sha, `sha-34ba9713`, and run the command.  Note, `sha-34ba9713` is the parachain version designation and this will change from time to time.  We will attempt to keep this documentation up to date but it may not always be.  Contact us on ________ for the latest sha to use.  If you see logs about a mismatched sha, this is the reason.  
+
+- [WIP] - this is stagenet - update for alphanet!!
+
+```
+docker run -d -p 30333:30333 -p 30334:30334 -v /var/lib/moonbase-alpha:/data/ \
+gcr.io/purestake-dev/moonbase-parachain-testnet:sha-34ba9713 \
+/moonbase-alphanet/moonbase-alphanet \
+    --base-path=/data \
+    --name="YOUR-NODE-NAME" \
+    --execution wasm \
+    --wasm-execution compiled \
+    -- \
+    --name="YOUR-NODE-NAME (Embedded Relay)"
+```
+
+If you want to expose WS or RPC ports, enable those on the docker run command line.  e.g. 
+   ```
+   docker run -d -p 30333:30333 -p 30334:30334 -p 9934:9934 -p 9945:9945 .... 
+   ```
+
+## Installation Instructions - Binary
+Moonbase Alphanet may be run on Unbuntu 18.04.  It may work with other flavors of linux, but Ubuntu is currently the only tested version.  
+
+Below are the steps to compile the binary and run moonbeam as a systemd service. 
+
+### Compiling the Binary 
+
+
+The following commands will build the latest release of the moonbeam parachain. 
+
+
+Clone the moonbeam repo.
+```
+cd ~
+git clone git@github.com:PureStake/moonbeam.git
+cd moonbeam
+```
+View the latest release .
+```
+git tag | tail -1
+```
+Checkout the latest release.
+```
+git checkout tags/$(git tag | tail -1)
+```
+Get the latest version of substrate.
+```
+curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+Run the init script.
+```
+./scripts/init.sh
+```
+Build the release.
+```
+cd ./node/parachain
```

Comment:
`cd`ing here is not necessary. You can run `cargo build --release` from the repo root. It builds the parachain by default.